README.md

```diff
@@ -0,0 +1,34 @@
+# Brainkeeper
+
+[![Build Status](https://drone.dev.sart.solutions/api/badges/brainkeeper/frontend/status.svg)](https://drone.dev.sart.solutions/brainkeeper/frontend)
+[![codecov](https://codecov.io/gh/brainkeeper/frontend/branch/master/graph/badge.svg)](https://codecov.io/gh/brainkeeper/frontend)
+[![Docker Pulls](https://img.shields.io/docker/pulls/hmbrainkeeper/frontend.svg)](https://hub.docker.com/r/hmbrainkeeper/frontend/)
+[![License: MIT](https://img.shields.io/badge/License-MIT-brightgreen.svg)](https://opensource.org/licenses/MIT)
+
+[**Visit the latest stable version**](https://app.brainkeeper.sart.solutions/)
+
+---
+
+### Introduction
+This Project was started by a [group of students](https://github.com/orgs/brainkeeper/people) from Munich University of Applied Sciences together with the Digital Transformation Lab and the local Alzheimer's Association
+
+### Overview
+Brainkeeper is a small specialized memory training app, which aims to improve, or slow the degeneration of the ability to remember and recognize faces and names.
+
+### Problem description
+A big problem of people with high age or Alzheimer’s often is, that they don’t recognize people like family members, old friends or medical personal they interact with on a regular basis. They often are ashamed because of that, and as a result of that stop socializing.
+
+### Idea
+Our Idea was to build a simple to use App, usable on all devices, where the Player can train there brain with faces of people who are of interest to him. 
+
+### Result
+We designed this simplistic App where the focus of the start screen is to just start playing. 
```

Comment:
App from the start screen on with a focus to simply start playing.

Reply:
I don’t like that, simplistic and simply in one sentence